spec/appendices.md

```diff
@@ -0,0 +1,33 @@
+# DRAFT Appendices
+
+## Security Considerations
+
+MessageFormat 2.0 _patterns_ are meant to allow a _message_ to include any string value
+which users might normally wish to use in their environment.
+Programming languages and other environments vary in what characters are permitted
+to appear in a valid string.
+In many cases, certain types of characters, such as invisible control characters,
+require escaping by these host formats.
+In other cases, strings are not permitted to contain certain characters at all.
+Since _messages_ are subject to the restrictions and limitations of their 
+host environments, their serializations and resource formats,
+that might be sufficient to prevent most problems.
+However, MessageFormat itself does not supply such a restriction.
+
+MessageFormat _messages_ permit nearly all Unicode code points,
+with the exception of surrogates, to appear in a _pattern_.
+This means that it can be possible for a _message_ to contain invisible unquoted
+control characters that abnormally affect the display of the _message_
```

Comment:
control characters that abnormally affect the display of the _message_
```suggestion
control characters (such as bidirectional format characters) that abnormally affect the display of the _message_
```

Reply:
@duerst's original comment was about C0 controls. The bidi controls also certainly affect the display. I'll cite both.